content/docs/command-reference/remote/modify.md

```diff
@@ -1162,6 +1146,22 @@ by HDFS. Read more about by expanding the WebHDFS section in
   $ dvc remote modify myremote ssl_verify path/to/ca_bundle.pem
   ```
 
+- `read_timeout` - set the time in seconds till a timeout exception is thrown
+  when attempting to read from a connection (60 by default). Let's set it to 5
+  minutes for example:
+
+  ```cli
+  $ dvc remote add myremote read_timeout 300
```

Comment:
@jorgeorpinel I'm not sure I understand this change, in order to change a parameter we need to use `modify`, not `add`. Is it something else that you wanted to modify?

Reply:
Sorry for the late reply here @dtrifiro! Yup, that was a typo on my end, thanks for fixing.